src/index.scss

```diff
@@ -1,9 +1,19 @@
-$feature-flags: (
```

Comment:
The `feature-flags` mapping should is needed. Your layout is not rendering properly since it has been deleted. See your preview https://carbon-tutorial-fpasqua1980.mybluemix.net/#/ and the expected layout in step 5 https://www.carbondesignsystem.com/tutorial/react-step-5

Reply:
Its fixed. Thanks